projects/demo-integrations/cypress/tests/demo-paths.ts

```diff
@@ -0,0 +1,89 @@
+export const DEMO_PATHS = [
```

Comment:
Would be great if we could automate this stuff. Can we reach for routing config?

Reply:
Now the paths are taken from the constant `pages` from demo with exclusions